packages/puppeteer/src/types.ts

```diff
@@ -0,0 +1,77 @@
+// AUTOGENERATED - Use `npm run generate:sources` to regenerate.
```

Comment:
This should not be committed. I think the file has been removed from the tree but accidentally pushed here.